eslintrc.json

```diff
@@ -24,6 +24,6 @@
     "prefer-const": "error",
     "sort-requires/sort-requires": "error",
     "strict": ["error", "global"],
-    "jest/no-identical-title": "warn"
+    "jest/prefer-to-have-length": "error"
```

Comment:
@thibaudcolas This doesn't look right, the `no-identical-title` rule here has been removed and in its place `prefer-to-have-length` was added.

We want both rules here to `error` don't we?

Reply:
Ah, thanks for he clarification 👍